jenkins/Jenkinsfile-release

```diff
@@ -3,6 +3,7 @@ library('jenkins-shared')
 
 mavenReleasePipeline(
     mavenVersion: 'Maven 3.6.x',
+    mavenSettingsFile: 'rsc-private-settings',
```

Comment:
-1 the ticket says to publish to central, and this IIUC would never end up there?

Reply:
> -1 the ticket says to publish to central, and this IIUC would never end up there?

Just reliased I misread the ticket 🙃